### DISCUSSION TITLE
Limit languages supported by Project in admin and Translations module

### DISCUSSION THREAD

**Discussion by VaZark** [directus/directus#11568] [Feature Requests]
i18n is a huge effort and adding a new language to support is a conscious choice each project has to make. So it would be helpful if the  Project settings tab had an option to limit the languages actively supported per project.

Sometimes it's easy to lose track of missing translations too. So a separate module for fields that have translations would be neat. IMO, Prestashop is a good reference. It has user-friendly translation support for both internal and module use.

**Comment by rijkvanzanten** [comment]
Could you elaborate on 

> Project settings tab had an option to limit the languages actively supported per project.

? 

When you're talking about translations, Directus by default relies on a regular collection, which in turn means you could simply add/remove whatever languages you want to that table, and control your translations interface accordingly. 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
